p2p/discover/table.go

```diff
@@ -24,27 +25,50 @@ type Table struct {
 	buckets [nBuckets]*bucket // index of known nodes by distance
 	nursery []*Node           // bootstrap nodes
 
+	bondmu    sync.Mutex
+	bonding   map[NodeID]*bondproc
```

Comment:
(I might figure this out as I go) What does this syntax mean?  Map of NodeId:bondproc?